docs/Product-Strategy/Roadmap-and-Backlog.md

```diff
@@ -1,50 +1,54 @@
 
-## Release 2: Early Secure Release
-This release will allow users to securely upload data into our system in production, replacing a less secure way of doing so, while increasing communication with the users and not increasing burden for OFA Admin staff. This release will allow approximately 8 tribes to pilot the use of login, upload, and download of files, while maintaining TDRS on the backend. While this type of workflow isn't our long term workflow, including it at this point delivers value to the users more quickly. 
 
-_Risks:_ The users who would be onboarded to this process would potentially have to learn and adjust with future releases and changes in functionality and workflow. Increased communication with them should help here.
-| Outcome | Status | 
-| -------- | ------- | 
-| Tribal users are engaged with OFA and communication channels are clear. | In Progress| 
-| Users know how to access and onboard TDP easily. | In Progress| 
-| Users can upload and transmit files securely to TDRS database. |Not Started|
-| Early Tribe and State users can securely upload reports with TDP instead of using unsecured email. |Not Started|
-| Users can resubmit files with error fixes |Not Started|
+| Outcome |
+| -------- | 
+| Tribal users are engaged with OFA and communication channels are clear. |
+| Users know how to access and onboard TDP easily. |
+| Users can upload and transmit files securely to TDRS database. |
+| Users can resubmit files with error fixes. |
+| Users will receive email notifications on the status of their account and data. |
+| Users can differentiate TANF and SSP files on the application. |
+| A research oriented TDP environment is created. |
+
 
 
 ## Release 3: Parity, Data Parsing, Automated Status, Notifications
-This release will parse and validate submitted data and store accepted data in a database. This workflow will include automated email notifications and user-friendly in-app error messages to help users better understand their data errors and submission history. This release will also include onboarding more users and will eventually deprecate the legacy TANF Data Reporting System.
+This release will parse and validate submitted data and store accepted data in a database. This workflow will include automated additional email notifications and user-friendly in-app error messages to help users better understand their data errors and submission history. This release will also include Django admin engancements and onboard all users as it will meet parity with the legacy TANF Data Reporting System.
```

Comment:
```suggestion
This release will parse and validate submitted data and store accepted data in a database. This workflow will include additional automated email notifications and user-friendly in-app error messages to help users better understand their data errors and submission history. This release will also include Django admin enhancements and onboard all users as it will meet parity with the legacy TANF Data Reporting System.
```